README.md

```diff
@@ -263,47 +261,45 @@ skate('my-element', {
       // This defaults to `false`. If you specify `true` the property name is
       // dash-cased and used as the attribute it should keep in sync with. If
       // you specify a `string`, it is used as the attribute name.
-      attr: false,
+      attribute: false,
 
-      // This property should be notified of changes when these dependencies
-      // are notified of changes.
-      //
-      // You may also specify dependencies as space-separated string.
-      deps: [
-        // This property's notify events will be triggered when its sibling
-        // `dependencyProperty` is notified of changes.
-        'dependencyProperty',
-
-        // This property's notify events will be triggered when the descendant's
-        // `deepDependencyProperty` notify events are triggered.
-        //
-        // At the time the dependency event bubbles up from the descendant, it
-        // must be accessible via `this.some.nested.child`. If it is not, then
-        // the dependency is ignored.
-        'some.nested.child.deepDependencyProperty'
-      ],
+      // Serializes the property value when it is set so that it can be stored
+      // as an attribute value. This only is called if this property is linked
+      // to an attribute.
+      serializeToAttribute: function () {},
```

Comment:
This option is named `serialize` in the implementation below (and ditto for `deserializeFromAttribute`).